terraform/eval_validate.go

```diff
@@ -101,6 +121,64 @@ func (n *EvalValidateProvisioner) Eval(ctx EvalContext) (interface{}, error) {
 	}
 }
 
+func (n *EvalValidateProvisioner) validateConnConfig(connConfig *ResourceConfig) (warns []string, errs []error) {
```

Comment:
🤦‍♂️ sorry, of course, I was worrying about the inverse.

Yeah, lets have this live here for now.